dapps/assets/src/i18n/de.json

```diff
@@ -1,12 +1,46 @@
 {
   "_assets": {
-    "contacts": "Kontakte",
-    "digitaltwins": "Digitale Zwillinge",
-    "digitaltwin": {
-      "digitaltwin-title": "Digitale Zwillinge",
-      "my-own": "Eigene",
+    "contacts": {
+      "add-contact-btn": "Kontakt hinzufügen",
+      "add-contact-desc": "Geben Sie die Evan-ID eines bestehenden Nutzers ein oder eine E-Mail-Adresse, um einen neuen Kontakt einzuladen.",
+      "add-contact-title": "Neuen Kontakt hinzufügen",
+      "all": "Alle",
+      "cancel": "Abbrechen",
+      "companies": "Unternehmen",
+      "contact-name-placeholder": "Max Mustermann",
+      "contact-name": "Kontaktname",
+      "contacts-empty": "Sie haben noch keine Kontakte hinzugefügt.",
+      "contacts-title": "Kontakte",
+      "created": "Erstellt",
+      "email-desc": "Der ausgewählte Nutzer wird mit Ihrer Anfrage kontaktiert. Im folgenden Formular können Sie diese bearbeiten.",
```

Comment:
actually no idea what the text should be, but this one sounds a bit strange... or? oO

Reply:
Im open for suggestions :D